bokeh/util/string.py

```diff
@@ -0,0 +1,57 @@
+""" Functions useful for string manipulations or encoding.
+
+"""
+from __future__ import absolute_import
+
+def encode_utf8(u):
+    """ Encode a UTF-8 string to a sequence of bytes.
+
+    Args:
+        u (str) : the string to encode
+
+    Returns:
+        bytes
+
+    """
+    import sys
```

Comment:
better move this import to the top of the file instead of repeating it inside functions